agent/agent.go

```diff
@@ -264,7 +263,8 @@ func initializeStateManager(cfg *config.Config, taskEngine engine.TaskEngine, cl
 		statemanager.AddSaveable("ContainerInstanceArn", containerInstanceArn),
 		statemanager.AddSaveable("Cluster", cluster),
 		statemanager.AddSaveable("EC2InstanceID", savedInstanceID),
-		statemanager.AddSaveable("ACSSeqNum", sequenceNumber),
+		//not tracking ACSSeqNum for now
```

Comment:
Any suggestions? I can change it to 

*ACSSeqNum is always pinned to '1'. In future we may want to deprecate this field completely from agent's state.*